addons/web/static/src/js/views/search_panel.js

```diff
@@ -33,7 +42,9 @@ const SEARCH_PANEL_DEFAULT_LIMIT = 200;
  */
 function _processSearchPanelNode(node, fields) {
     const sections = {};
-    node.children.forEach(({ attrs, tag }, index) => {
+    const incompatibilities = { category: null, filter: null };
```

Comment:
Suggestion (I think it is simpler to understand):
`let hasFilterWithDomain = false;`